test/baggage/default_baggage_restriction_manager.js

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+import {assert} from 'chai';
+import DefaultBaggageRestrictionManager from "../../src/baggage/default_baggage_restriction_manager";
+import Restriction from "../../src/baggage/restriction";
+
+describe('DefaultBaggageRestrictionManager should', () => {
+
+    it ('return a Restriction', (done) => {
+        let mgr = new DefaultBaggageRestrictionManager();
+        assert.deepEqual(mgr.getRestriction("key"), new Restriction(true, 2048));
```

Comment:
I believe for things like this we would want it to break because what you're suggesting is tantamount to this.var == this.getVar.

I remember there being a informative Google Testing Blog entry about this but can't find it.